### PR TITLE
Readme changes, github actions ci, test

### PR DESCRIPTION
**Pull request by ethteck**
Adds a simple Github Actions build and test
Changes STDFLAGS in Makefile to use `-std=c++2a` for older GCC compatibility
Added error handling for nonexistant input file
Minor README.md edits

### CODE REVIEW

**Comment by ethteck** [comment]
test (yo lol)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  build_and_test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install GCC multilib
+        run: |
+          sudo apt update
+          sudo apt install gcc-multilib g++-multilib
```
Comment (mkst) [inline .github/workflows/ci.yml:13]: does this need a `-y` ?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
 # WiBo
 
-An experiment to try and write a minimal, low-fuss wrapper that can run really simple command-line 32-bit Windows binaries on Linux - with less faff and less dependencies than WINE.
+A minimal, low-fuss wrapper that can run really simple command-line 32-bit Windows binaries on Linux - with less faff and less dependencies than WINE.
 
 Don't run this on any untrusted executables, I implore you. (Or probably just don't run it at all... :p)
 
     make && ./wibo
 
-If you need something like this project (but more mature), you might find [taviso/loadlibrary](https://github.com/taviso/loadlibrary) more interesting.
```
Comment (simonlindholm) [inline README.md:9]: I found this reference useful/interesting fwiw

File: main.cpp [hunk 3]
```diff
@@ -176,7 +177,20 @@ int main(int argc, char **argv) {
...
+
+	if (!std::filesystem::exists(pe_path)) {
+		printf("File not found: %s\n", pe_path);
+		return 1;
+	}
+
+	FILE *f = fopen(pe_path, "rb");
+
+	if (!f) {
+		perror("Failed to open file\n");
```
Comment (simonlindholm) [inline main.cpp:190]:
```suggestion
                std::string mesg = std::string("Failed to open file ") + pe_path;
		perror(mesg.c_str());
```

File: main.cpp [hunk 4]
```diff
@@ -176,7 +177,20 @@ int main(int argc, char **argv) {
...
 	wibo::mainModule = &exec;
 
-	FILE *f = fopen(argv[1], "rb");
+	char* pe_path = argv[1];
+
+	if (!std::filesystem::exists(pe_path)) {
+		printf("File not found: %s\n", pe_path);
+		return 1;
+	}
+
```
Comment (simonlindholm) [inline main.cpp:186]:
```suggestion
```
(covered by the below, right?)

File: test/test.c [hunk 5]
```diff
@@ -0,0 +1,8 @@
+float apple = 3.0f;
+float banana = 65.32f;
+
+int something(void) {
+    return (int)(apple * banana) % 11;
+}
+
+something();
```
Comment (simonlindholm) [inline test/test.c:8]: mistake?

**Review by simonlindholm — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ethteck) [reply README.md:9]: I'll add a related projects section at the bottom

File: test/test.c (hunk 5 above)
Comment (ethteck) [reply test/test.c:8]: no, this file exists for the sake of testing. or do you just mean the function call at the end?

Comment (simonlindholm) [reply test/test.c:8]: yeah, the legacy-style function declaration at the end

File: main.cpp (hunk 4 above)
Comment (simonlindholm) [reply main.cpp]: There are, but if the file is not found fopen will still fail and perror will append a cause like "file not found" to the error

**Pull request merged**